nes-sources/private/FileSource.hpp

```diff
@@ -70,7 +70,7 @@ struct ConfigParametersCSV
         { return NES::Configurations::DescriptorConfig::tryGet(FILEPATH, config); }};
 
     static inline std::unordered_map<std::string, NES::Configurations::DescriptorConfig::ConfigParameterContainer> parameterMap
-        = NES::Configurations::DescriptorConfig::createConfigParameterContainerMap(FILEPATH);
+        = NES::Configurations::DescriptorConfig::createConfigParameterContainerMap(SourceDescriptor::parameterMap, FILEPATH);
```

Comment:
What about the `TCPSource` in `nes-plugins` or the sinks? Change it for all implementations or for none